front/src/modules/metadata/hooks/useSeedCustomObjectsTemp.ts

```diff
@@ -1,83 +1,45 @@
-import {
-  CreateOneFieldMutation,
-  CreateOneFieldMutationVariables,
-  CreateOneObjectMutation,
-  CreateOneObjectMutationVariables,
-} from '~/generated-metadata/graphql';
-
-import { CREATE_ONE_FIELD, CREATE_ONE_OBJECT } from '../graphql/mutations';
-
-import { useApolloClientMetadata } from './useApolloClientMetadata';
+import { useCreateOneMetadataField } from './useCreateOneMetadataField';
+import { useCreateOneMetadataObject } from './useCreateOneMetadataObject';
 
 export const useSeedCustomObjectsTemp = () => {
-  const client = useApolloClientMetadata();
+  const { createOneMetadataObject } = useCreateOneMetadataObject();
```

Comment:
can we integrate this with existing seeds instead? This is introducing a new way of seeding

Reply:
and IMO, the frontend should not be the one seeding